app/assets/javascripts/ckeditor/plugins/cortex_media_insert/plugin.js

```diff
@@ -1,19 +1,19 @@
-(function (global) { 
+(function (global) {
   'use strict';
 
   global.CKEDITOR.plugins.add('cortex_media_insert', {
     icons: 'media',
     init: function (editor) {
       editor.addCommand('insertMedia', {
         exec: function (editor) {
-          window.MODALS.featured.open();
+          window.MODALS.wysiwyg.open();
```

Comment:
why rename this line and line 16 from 'featured' or 'media' to the more general 'wysiwyg'?